### DISCUSSION TITLE
BLS12377 on BN254

### DISCUSSION THREAD

**Discussion by LautaroLasorsa** [Consensys/gnark#713] [Q&A]
Hello everyone, I hope you have a nice day.

I am trying to compile the BLS 12377 sample with the Curve BN254, but I get the error "outer curve  21888242871839275222246405745257275088548364400416034343698204186575808495617 does not have a inner curve". Is there any way to solve it? Or is it impossible to compile BLS 12377 with the Curve BN254?

Thanks in advance.

**Comment by ivokub** [comment]
Hi. Can you provide minimal runnable example? As a Go test would be best.

**Reply by LautaroLasorsa** [reply]
Thanks for the answer, but I have already solved it with the BN 254 emulated.

Sorry for the delay.
